lib/option_converter.js

```diff
@@ -137,12 +137,13 @@ var contentFormatToString = function(value) {
   else if (value.length === 2)
     value = value.readUInt16BE(0)
   else
-    throw new Error('Content-Format option is too big')
+    return value    // the application is still responsible to validate the value
+    //throw new Error('Content-Format option is too big')
 
   var result = formatsBinaries[value]
 
-  if (!result)
-    throw new Error('No matching format found')
+  if (!result) return value
```

Comment:
I am wondering if it makes sense to return the value here or if it would be better to return `null` since a value that is longer than two bytes is prohibited by the RFC. Using `null` would probably make the handling of errors easier as well I assume?

Reply:
After thinking about this, returning the numeric value in this very case actually makes a lot of sense to ensure that all messages can be handled. However, maybe returning `null` when the content-format is too big could be an option?